### DISCUSSION TITLE
Custom widgets from Allure dashboard are not clickable

### DISCUSSION THREAD

**Discussion by natalliachu** [allure-framework/allure2#1944] [TestOps]
Hi Allure team,

I'd like to submit a bug: I’ve created a new Allure dashboard and added two widgets there: 
-     the number of tests based on ownership owning (test case query is role["Owner"]="@team_name")
-     the number of muted tests we’re owning (test case query is muted=true and role["Owner"]="@team_name")
both have Test Case Pie Chart type. 

The widgets were added successfully, the numbers on the widgets are correct and they look clickable **but when I click on them - nothing happened.**

**What I’d expect:** the list of test cases filtered by the widget queue will be opened. I know such functionality exists because other widgets on our Overview dashboard are clickable - the list of filtered test cases are opened by clicking on widgets.

Thank you!

**Comment by cheshi-mantu** [comment]
@natalliachu hi there
this is not a bug if you are talking about Allure TestOps.
Not all widgets support click and filter actions. 

Please consider using https://help.qameta.io for the issues related to Allure TestOps. You need to use the corporate email when registering there.

**Comment by natalliachu** [comment]
Thank you, @cheshi-mantu!  As you mentioned [here](https://help.qameta.io/support/tickets/2618) it could be converted into feature request. I will stay updated with that ticket so this discussion here in Github can be closed.

**Reply by ar6aaz** [reply]
Hey @natalliachu was there an update on your ticket? I can't access the link. I've logged a similar support ticket but haven't received any response from TestOps support.

**Reply by natalliachu** [reply]
@ar6aaz  no, unfortunately I also didn't receive any updates yet.
